package.json

```diff
@@ -81,6 +82,7 @@
 		"postcss-loader": "^7.3.3",
 		"prettier": "^2.8.8",
 		"prop-types": "^15.8.1",
+		"re-resizable": "^6.9.9",
```

Comment:
Shouldn't these be under `dependencies` as they are not being used for development only? Also since they are being used for the extension package, they should be added to `packages/extension/package.json` instead